WooCommerce/Classes/POS/Models/PointOfSaleAggregateModel.swift

```diff
@@ -194,7 +195,9 @@ extension PointOfSaleAggregateModel {
 
     @MainActor
     private func collectCardPayment() async {
-        guard let order = orderController.order else {
+        guard case let .loaded(totals, order) = internalOrderState,
```

Comment:
That's a nice one, I didn't realized that we have now the Order as second associated value to .loaded state.

Reply:
Yeah, only internally, I don't want it leaking out to the view. I hadn't realised we could remove the `order` dependency either, and I wrote botho of them in!